eth/scripts/upgrade_bridge_live.js

```diff
@@ -0,0 +1,28 @@
+const fs = require("node:fs");
+const { ethers } = require("hardhat");
+
+async function main() {
+  const signers = await ethers.getSigners();
+  const accounts = signers.map((s) => s.address);
+
+  console.log("Using", accounts[0], "as the transaction signer");
+
+  const Most = await ethers.getContractFactory("Most");
+  console.log("Deploying Most...");
+
+  const most = await Most.deploy();
+  console.log("Upgrade for Most deployed to:", most.target);
+
+  addresses = { most: most.target };
```

Comment:
This will be confusing - one would expect address of the proxy here, and not of the logic contract